### DISCUSSION TITLE
Обращение к полю, определённому строкой кода

### DISCUSSION THREAD

**Discussion by michael-sosin** [totumonline/totum-ru-issues-and-discussions#628] [Как это сделать?] [answered]
Здравствуйте. Вопрос по документации из раздела https://ru.docs.totum.online/syntax#sx-addressing-field-by-line.

> Возможно обращение к полю, определенному строкой кода при помощи #$whatfield. Таким образом значение в строке whatfield: будет определять name поля из которого брать данные.

Задача: взять name поля из ячейки в другой таблице для подстановки в код действия таким образом:

`= : set(table: $#ntn; field: $#nf = #$fieldname; where: 'id' = #id)`

Вопрос: какое значение нужно передать в переменную `$field`, чтобы задать имя поля с доступом к значениям JSON, например, имя поля должно быть `#fieldname[content][folder1]`.

Код работает только таким образом `$fieldname: "fieldname2"`.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
```
=: set(table: $#ntn; field: $#nf = #$fieldname[content][folder1]; where: 'id' = #id)

fieldname: "some_field_name"
```

**Reply by michael-sosin** [reply]
Помогло. Спасибо!
